examples/spring-boot-kotlin/src/main/java/example/springframework/boot/minimal/Main.kt

```diff
@@ -0,0 +1,14 @@
+package example.springframework.boot.minimal
+
+import org.springframework.boot.SpringApplication
+import org.springframework.boot.autoconfigure.SpringBootApplication
+
+@SpringBootApplication
+open class Main {
```

Comment:
Spring is guiding you to [a better example](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-kotlin.html#boot-features-kotlin-api-runapplication) than the change from java to kotlin function provided by idea.

Wouldn't this be more familiar to others?